README.md

```diff
@@ -174,67 +182,52 @@ wrapping a component you intend to style, you'll need to make sure you accept
 the `className` as a prop and apply it to where you want the styles applied in
 your custom component (normally the root element).
 
+```javascript
+const UnstyledComp = ({ className, children }) => <div className={`${className} other-class`>{children}</div>;
+const MyStyledComp = glamorous(UnstyledComp)({ margin: 1 });
```

Comment:
I'm just too used to that 🙈 https://github.com/braposo/glamorous/commit/cbbde3016e342899d014279688aaeb7d32a8d94b

Also found a couple of other places with semis so removed them as well.